src/app/controller/openTaskRegisterModal.ts

```
@@ -0,0 +1,15 @@
+import { makeTaskRegisterBlock } from "@/app/SlackBlocks";
+import { postViewsOpen } from "@/Slack/api";
+
+/**
+ * 依頼登録モーダルを表示する
+ * @param {any} payload - ペイロード
+ */
+export function openTaskRegisterModal(
+  payload: any,
+): GoogleAppsScript.Content.TextOutput {
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)